src/test/resources/IUDX-Resource-Server-Consumer-APIs-V4.0.postman_collection.json

```diff
@@ -8959,6 +9033,77 @@
 						}
 					},
 					"response": []
+				},
+				{
+					"name": "200 (success) multi attribute",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"exec": [
+									"// Test the response code",
+									"pm.test(\"response is 200 (OK)\", function () {",
+									"    pm.response.to.have.status(200);",
+									"});",
+									"",
+									"// Test the response header",
+									"pm.test(\"Check response header\", function () {",
+									"    pm.response.to.have.header(\"Content-Type\",\"application/json\");",
+									"});",
+									"",
+									"// Test the response",
+									"pm.test(\"Check response body\", function () {    ",
+									"    const body = pm.response.json();",
+									"    pm.expect(body).to.have.property(\"title\", \"Success\");",
+									"    const resultsjsonData = body.results[0];",
+									"    pm.expect(resultsjsonData).to.have.property(\"count\", 957);",
```

Comment:
Please correct the postman assertion test, the API response for count will not have count as property 

```json
{
    "type": "urn:dx:rs:success",
    "title": "Success",
    "results": [
        {
            "totalHits": 19
        }
    ]
}

```

Reply:
done